<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="input_field"/>
        </item>
        <item>
         <widget class="QPushButton" name="search_button">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Select file -&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchFileButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeBox">
          <item>
           <property name="text">
            <string>car</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>truck</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bus</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777176</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Brand:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputBrand"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Model:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputModel"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Year:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputYear"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorbox">
          <item>
           <property name="text">
            <string>red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>purple</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Other:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputOther"/>
        </item>
        <item>
         <widget class="QPushButton" name="addbutton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="customBrowser" name="output_field">
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customBrowser</class>
   <extends>QTextBrowser</extends>
   <header>custombrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>search_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchcar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>input_field</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addcar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputBrand</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputModel</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>858</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputYear</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputOther</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchcar()</slot>
  <slot>addcar()</slot>
  <slot>selectFile()</slot>
 </slots>
</ui>
